tests/tracking/test_client.py

```diff
@@ -477,3 +486,33 @@ def test_create_model_version_copy_not_called_to_nondb(mock_registry_store):
 
 def _default_model_version():
     return ModelVersion("model name", 1, creation_timestamp=123, status="READY")
+
+
+def test_get_databricks_runtime_nondb(mock_spark_session):
+    runtime = get_databricks_runtime()
+    assert runtime is None
+    mock_spark_session.conf.get.assert_not_called()
+
+
+def test_get_databricks_runtime_no_spark_session():
+    with mock.patch(
+        "mlflow.utils.databricks_utils._get_active_spark_session", return_value=None
+    ), mock.patch("mlflow.utils.databricks_utils.is_in_databricks_notebook", return_value=True):
+        runtime = get_databricks_runtime()
+        assert runtime is None
+
+
+def test_get_databricks_runtime_in_notebook(mock_spark_session):
+    with mock.patch("mlflow.utils.databricks_utils.is_in_databricks_notebook", return_value=True):
+        get_databricks_runtime()
+        mock_spark_session.conf.get.assert_called_once_with(
+            "spark.databricks.clusterUsageTags.sparkVersion", default=None
```

Comment:
What does `default = None` do here?

Reply:
It seems like `assert_called_once_with` also checks that keyword arguments match when making the call. When I called, `spark.conf.get` in `get_databricks_runtime`, if no default value was provided, the get call would throw an exception, so `default=None` just makes sure that a safe get is used.

Reply:
Makes sense!